dojo/db_migrations/0193_alter_usercontactinfo_slack_username.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 4.1.11 on 2023-11-22 23:45
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("dojo", "0192_notifications_scan_added_empty"),
+    ]
+
+    def replace_slack_email_with_channel_id(apps, schema_editor):
+        UserContactInfo = apps.get_model('dojo', 'UserContactInfo')
+        for obj in UserContactInfo.objects.all():
+            obj.slack_username = obj.slack_user_id
+            obj.save()
+
+    def unreplace_slack_email_with_channel_id(apps, schema_editor):
+        UserContactInfo = apps.get_model('dojo', 'UserContactInfo')
+        for obj in UserContactInfo.objects.all():
+            obj.slack_user_id = obj.slack_username
+            obj.save()
+
+    operations = [
+        migrations.AlterField(
+            model_name="usercontactinfo",
+            name="slack_username",
+            field=models.CharField(
+                blank=True,
+                help_text="Slack channel name (with optional # prefix) or ID",
+                max_length=150,
+                null=True,
+                verbose_name="Slack channel",
+            ),
+        ),
+        migrations.RunPython(code=replace_slack_email_with_channel_id,
+                             reverse_code=unreplace_slack_email_with_channel_id),
+        migrations.RemoveField(
+            model_name="usercontactinfo",
+            name="slack_user_id",
+        ),
+    ]
```

Comment:
What about
```
 migrations.RemoveField(
            model_name="usercontactinfo",
            name="slack_username",
        ),
 migrations.RenameField(
            model_name="usercontactinfo",
            old_name="slack_user_id",
            new_name="slack_username",
        ),
```
Looks easier compare to `RunPython`
https://docs.djangoproject.com/en/5.0/ref/migration-operations/#renamefield

Reply:
Indeed, thanks. And then `AlterField` needs to be moved to the end, to set up `verbose_name` etc.

Moreover, I should set `max_length=81` ([`#` plus 80 characters of the channel name](https://api.slack.com/methods/conversations.rename#naming)).